tests/python/test_python_frontend.py

```diff
@@ -4458,6 +4457,140 @@ def fusion_func(fd: FusionDefinition):
         for i in range(num_out):
             self.assertEqual(nvf_out[i].data_ptr(), inputs[0].data_ptr())
 
+    @unittest.skipIf(is_pre_ampere(), "Only supported on Ampere and newer devices.")
+    def test_sdpa_fwd(self):
```

Comment:
I realize this test is long. Once https://github.com/NVIDIA/Fuser/pull/2701 goes through, I intend to move sdpa tests to a separate file.